### ISSUE TITLE
Allow `alert()` messages

### ISSUE THREAD

**Issue by crhallberg** [processing/p5.js-web-editor#507]
I've noticed that you cannot use `alert()` inside of sketches. This seems to be due to the sandboxing of the iframe and the lack of `allow-modals` which has been referred to in previous issues ([#2360](https://github.com/processing/p5.js/issues/2360)).

I'm assuming this is intentionally omitted to prevent printing, but based on what I read in [Issue 474](https://github.com/processing/p5.js-web-editor/issues/474#issuecomment-344036753) using `print()` outside of functions to actually print is the intended design. With `allow-modals`, print is still successfully sandboxed inside of functions and alerts work properly.

I can make a pull request, but I wanted to make sure this was a good direction to go in before I do so.

**Comment by catarak** [comment]
makes sense to me to remove this from the sandboxing of the `iframe`. go ahead and change it!

**Comment by awarua** [comment]
Should this issue be closed following the merge of crhallberg's commit?
